app/src/main/java/org/wikipedia/feed/announcement/AnnouncementCardView.java

```diff
@@ -83,6 +84,7 @@ public AnnouncementCardView(@NonNull Context context) {
 
         if (card.hasFooterCaption()) {
             footerTextView.setText(StringUtil.fromHtml(card.footerCaption()));
+            RichTextUtil.removeUnderlinesFromLinks(footerTextView);
```

Comment:
@cooltey Line #93 the border condition needs to be taken care of. We had thick red borders previously. We need some solution now.

Reply:
Also, the space between title and text seems to be a little bit more, since we are doing this in the same string, may be use one less break <br>?

Reply:
Also, the text size is different.20sp.

Reply:
> Also, the space between title and text seems to be a little bit more, since we are doing this in the same string, may be use one less break <br>?
> Also, the text size is different.20sp.

Thanks, all the questions are waiting for Robin's feedback: https://phabricator.wikimedia.org/T263482#6519398

Reply:
> 
> 
> @cooltey Line #93 the border condition needs to be taken care of. We had thick red borders previously. We need some solution now.

Done.